apps/client/pages/settings/index.tsx

```diff
@@ -66,6 +67,10 @@ function HowItWorks() {
                 Twitcaster ignores casts that are replies to another cast. I.e.
                 it will only tweet top-level casts.
               </Text>
+              <Text>
+                If the <code>#twitcast</code> hashtag option is toggled on,
```

Comment:
What do you think of shortening this the hashtag to `#twt`?

Reply:
both?

Reply:
but only document one explicitly 😄